relayer/relays/beacon/header/header.go

```diff
@@ -63,6 +60,15 @@ func (h *Header) Sync(ctx context.Context, eg *errgroup.Group) error {
 	h.cache.SetInitialCheckpointSlot(lastFinalizedHeaderState.InitialCheckpointSlot)
 	h.cache.AddCheckPointSlots([]uint64{lastFinalizedHeaderState.BeaconSlot})
 
+	// Special handling here for the initial checkpoint to sync the next sync committee which is not included in initial
+	// checkpoint.
+	if h.isInitialSyncPeriod() {
+		err = h.SyncCommitteePeriodUpdate(ctx, latestSyncedPeriod)
+		if err != nil {
+			return fmt.Errorf("sync next committee for initial sync period: %w", err)
+		}
+	}
```

Comment:
I moved this special case earlier. [This check](https://github.com/Snowfork/snowbridge/blob/main/relayer/relays/beacon/header/header.go#L172 ) will cause the initial checkpoint's next sync committee only to be synced when the period rolls over. I think it is better to sync it immediately when it is available.